source/VMelnalksnis.PaperlessDotNet/Documents/DocumentContent.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 Valters Melnalksnis
+// Licensed under the Apache License 2.0.
+// See LICENSE file in the project root for full license information.
+
+using System.IO;
+
+namespace VMelnalksnis.PaperlessDotNet.Documents;
+
+/// <summary>
+/// The content of a document.
+/// </summary>
+public sealed class DocumentContent
```

Comment:
I'd suggest implementing `IDisposable` through the `Content` property, so that the stream is disposed.
```suggestion
public sealed class DocumentContent : IDisposable
{
	/// <inheritdoc />
	public void Dispose() => Content.Dispose();
```